google-account-plugin/testSrc/com/google/cloud/tools/intellij/login/CredentialedUserRosterTest.java

```diff
@@ -47,8 +43,8 @@ public void setUp() {
   }
 
   /**
-   * Tests that {@link com.google.intellij.login.Users#addUser(CredentialedUser)}
-   * stores the proper users and currently manages the active user.
+   * Tests that {@link CredentialedUserRoster#addUser(CredentialedUser)} stores the proper users and
```

Comment:
Are these comments necessary? Typically there are no javadoc comments on tests; it should be easily readable so the reader knows what it's testing.

Reply:
true. I wasn't planning on refactoring the existing code except for the broken reference. but I agree these are redundant.